### PR TITLE
fix: updated project table as per pervious task's project table

### PR DESCRIPTION
**Pull request by meshramsaroj**
**Task link:** https://bloomstack.com/desk#Form/Task/TASK-2021-00712

**GIF:**

dependent-task-project-table

### CODE REVIEW

File: erpnext/projects/doctype/task/task.js [hunk 1]
```diff
@@ -38,6 +38,16 @@ frappe.ui.form.on("Task", {
 				filters: filters
 			};
 		})
+
+		let prev_route = frappe.get_prev_route();
+		if (prev_route && prev_route[1] === 'Task') {
```
Comment (sahil28297) [inline erpnext/projects/doctype/task/task.js]: @meshramsaroj Will this clear out the already existing fields in child table? please check. If it does, add a condition that the child table should be empty. something like
```suggestion
		if (prev_route && prev_route[1] === 'Task')  && (!frm.doc.projects){
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Review by neilLasrado — CHANGES_REQUESTED** [review]
Add a condition to check if the table is empty & set only if empty

File: erpnext/projects/doctype/task/task.js (hunk 1 above)
Comment (meshramsaroj) [reply erpnext/projects/doctype/task/task.js]: @sahil28297  fixed changes.
Please check

**Comment by sahil28297** [comment]
@meshramsaroj These changes will go in current sprint branch, oct-sprint-1

**Comment by sahil28297** [comment]
> @meshramsaroj These changes will go in current sprint branch, oct-sprint-1

@meshramsaroj please open a new PR with the base branch as oct-sprint-1 and close this PR

**Comment by meshramsaroj** [comment]
closing this PR. created new PR from oct-sprint-1 branch
PR link: https://github.com/Bloomstack/erpnext/pull/1840

**Pull request closed without merging**